pkg/progressbar/print.go

```diff
@@ -47,10 +48,23 @@ func Print(progress <-chan client.ImageProgress) error {
 				continue
 			}
 
+			if update.CurrentTask != "" && update.CurrentTask != currentTask {
+				if bar != nil {
+					bar.Add(bar.Total - bar.Current)
+					_, _ = bar.Stop()
+					bar = nil
+				}
+				currentTask = update.CurrentTask
+			}
```

Comment:
Here, we reset the progress bar if the current task has changed, since we need to start a new bar for the new task.